src/map_command_handler.hpp

```diff
@@ -359,6 +387,11 @@ class map_command_handler
 	{
 		cmd_prefix_ = value;
 	}
+	// set command or message flag
```

Comment:
There's no documentation about what the "command or message" flag means. Please document how this method changes the observable behavior.

Reply:
Hope it is now fine. Could be shorter, but I like explaining thoroughly.